### PR TITLE
Current state Quality Declaration, Contributing and Readme files

### PR DESCRIPTION
**Pull request by Blast545**
This PR adds the current Quality Declaration claim for the rcl_logging_spdlog package. The current state is Quality level 4.

It will also add Contributing.md file to properly document License contributions and Readme pointing to current Quality Declaration.

Signed-off-by: Jorge Perez <jjperez@ekumenlabs.com>

### CODE REVIEW

**Comment by Blast545** [comment]
Reformated to match new REP-2004 format

File: rcl_logging_spdlog/Quality_Declaration.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+This document is a declaration of software quality for the `rcl_logging_spdlog` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
```
Comment (brawner) [inline rcl_logging_spdlog/Quality_Declaration.md:1]: Rename this file appropriately.

File: rcl_logging_spdlog/Quality_Declaration.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+### Change Requests [2.i]
+
+All changes will occur through a pull request, check [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.
+
+### Contributor Origin [2.ii]
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](./CONTRIBUTING.md)
+
+### Peer Review Policy [2.iii]
+All pull requests will be peer-reviewed, check [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.
```
Comment (brawner) [inline rcl_logging_spdlog/Quality_Declaration.md:50]: Following the recommended guidelines in the [ROS 2 Developer Guide] all pull requests must have at least 1 peer review.

File: rcl_logging_spdlog/Quality_Declaration.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+`rcl_logging_spdlog` does not include tests, so coverage is not provided.
+
+### Performance [4.iv]
+
+`rcl_logging_spdlog` does not conduct performance tests.
+
+### Linters and Static Analysis [4.v]
+
+`rcl_logging_spdlog` uses and passes all the standard linters and static analysis tools for a C+ package as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters-and-static-analysis). Passing implies there are no linter/static errors when testing against CI of supported platforms.
```
Comment (brawner) [inline rcl_logging_spdlog/Quality_Declaration.md:108]: for a C package

File: rcl_logging_spdlog/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# rcl_logging_spdlog
+
+Package supporting an implementation of logging functionality using `spdlog`.
+
+This package claims to be in the **Quality Level 4** category, see the [Quality Declaration](./Quality_Declaration.md) for more details.
```
Comment (brawner) [inline rcl_logging_spdlog/README.md:5]: This needs the ## header

File: rcl_logging_spdlog/Quality_Declaration.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+`rcl_logging_spdlog` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process).
+
+### Change Requests [2.i]
+
+All changes will occur through a pull request, check [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change-control-process) for additional information.
+
+### Contributor Origin [2.ii]
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](./CONTRIBUTING.md)
```
Comment (ahcorde) [inline rcl_logging_spdlog/Quality_Declaration.md:47]: There is no contributing file

File: rcl_logging_spdlog/Quality_Declaration.md (hunk 2 above)
Comment (ahcorde) [inline rcl_logging_spdlog/Quality_Declaration.md:50]: newline

File: rcl_logging_spdlog/Quality_Declaration.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+`spdlog_vendor` was declared to be Quality Level 4 [here](https://github.com/ros2/spdlog_vendor/blob/master/Quality_Declaration.md).
+
+### Optional Direct Runtime ROS Dependencies [5.ii]
+
+`rcl_logging_spdlog` has no optional Direct Runtime ROS dependencies that need to be considered for this declaration.
+
+### Direct Runtime non-ROS Dependency [5.iii]
+
+`rcl_logging_spdlog` has a Direct Runtime non-ROS dependenciy the `spdlog` library. It was declared to be Quality Level 4 [here](https://github.com/ros2/spdlog_vendor/blob/master/SPDLOG_QUALITY_DECLARATION.md).
+
```
Comment (ahcorde) [inline rcl_logging_spdlog/Quality_Declaration.md:134]: extra line

File: rcl_logging_spdlog/Quality_Declaration.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+## Platform Support [6]
+
+`rcl_logging_spdlog` supports all of the tier 1 platforms as described in [REP-2000](https://www.ros.org/reps/rep-2000.html#support-tiers), and tests each change against all of them.
+
+## Vulnerability Disclosure Policy [7.i]
```
Comment (ahcorde) [inline rcl_logging_spdlog/Quality_Declaration.md:140]:
```suggestion
## Security [7]

### Vulnerability Disclosure Policy [7.i]
```

**Pull request merged**